dev/io.openliberty.jcache.internal/src/io/openliberty/jcache/internal/Activator.java

```diff
@@ -82,7 +85,8 @@ public void removedService(ServiceReference<Condition> ref, Boolean present) {
 
     @Override
     public void configurationEvent(ConfigurationEvent event) {
-        if ("io.openliberty.jcache.cachemanager".equals(event.getFactoryPid())
+        Set<String> pids = new HashSet<>(Arrays.asList("io.openliberty.jcache.cachemanager", "io.openliberty.jcache.cachingprovider"));
```

Comment:
This should be a static final `Set<String>`

Reply:
made it final, thanks!

Reply:
But it should also be a static constant.  The final var is currently re-created each call to `configureEvent`.